data/json/mapgen_palettes/stream.json

```
@@ -0,0 +1,44 @@
+[
+  {
+    "type": "palette",
+    "id": "stream_palette",
+    "terrain": {
+      " ": "t_null",
+      "~": "t_water_moving_sh",
+      "1": [
+        [ "t_null", 100 ],
+        [ "t_sand", 1 ],
+        [ "t_clay", 4 ],
+        [ "t_grass_long", 900 ],
+        [ "t_region_shrub", 20 ],
+        [ "t_region_tree", 20 ]
+      ],
+      "2": [ [ "t_null", 150 ], [ "t_grass_long", 10 ], [ "t_region_shrub", 1 ], [ "t_region_tree", 1 ] ]
+    },
+    "furniture": {
+      " ": "f_null",
+      "1": [
+        [ "f_null", 800 ],
+        [ "f_region_weed", 10 ],
+        [ "f_boulder_small", 3 ],
+        [ "f_boulder_medium", 2 ],
+        [ "f_boulder_large", 1 ]
+      ],
+      "2": [
+        [ "f_null", 1400 ],
+        [ "f_region_weed", 10 ],
+        [ "f_boulder_small", 3 ],
+        [ "f_boulder_medium", 2 ],
+        [ "f_boulder_large", 1 ]
+      ]
+    },
+    "items": {
+      "1": [
+        { "item": "trash", "chance": 2, "repeat": [ 0, 2 ], "damage": [ 0, 4 ] },
+        { "item": "creek_bed", "chance": 15, "repeat": [ 0, 2 ] },
+        { "item": "wheels", "chance": 1, "damage": [ 2, 5 ] }
+      ],
+      "2": [ { "item": "trash", "chance": 2, "repeat": [ 0, 2 ] }, { "item": "creek_bed", "chance": 10, "repeat": [ 0, 3 ] } ]
```

None of these need to spam rolls for items so aggressively to be honest when they're being rolled constantly for multiple tiles of a long winding overmap special. Also the trash spawn is damaged in one tile but undamaged in the other, when basically no other wilderness trash spawns pre-damaged? 
```suggestion
        { "item": "trash", "chance": 1 },
        { "item": "creek_bed", "chance": 1 },
        { "item": "wheels", "chance": 1, "damage": [ 2, 5 ] }
      ],
      "2": [ { "item": "trash", "chance": 1 }, { "item": "creek_bed", "chance": 1 } ]
```